src/coresyf_image_crop/coresyf_image_crop.py

```diff
@@ -76,6 +77,40 @@ def get_shapefile_crs(data_source):
     return epsg_code
 
 
+def crop_raster(input_raster, output_raster_file, polygon_extent, dest_crs):
+    '''
+    Crops the image specified by input_raster using the limits defined in
+    polygon_extent (buffer already applied).
+
+    input_raster: path to the image to be cropped.
+    output_raster_file: the output file path.
```

Comment:
Both variables are paths, so we can try to give them similar names such as: 
1. "input_raster_path" and "output_raster_path" or
2. "input_path" and "output_path"